tm2py/model/demand/air_passenger.py

```diff
@@ -0,0 +1,113 @@
+"""TODO: review module docsting 
+Build the airport trip matrices (these are independent of level-of-service)
+    Run the airport model
+        "nonres/BuildAirPax.job"
+            input: nonres/@token_year@_@token_toFrom@@token_airport@.csv
+            output: five time-of-day-specific tables for DA, SR2, SR3, DATOLL, SR2TOLL, SR3TOLL
+            Looks like a pandas application
+
+NOTES FROM BuildAirPaxMatrices.job
+
+ TP+ script that creates air passenger vehicle trip tables for the Bay Area's three major airports, namely SFO,
```

Comment:
Here and lots of other places your lines are too long: max should be 99 characters.

Run Flake8 to identify.